WordPress/src/main/java/org/wordpress/android/ui/stats/models/InsightsAllTimeModel.java

```diff
@@ -0,0 +1,65 @@
+package org.wordpress.android.ui.stats.models;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.Serializable;
+
+public class InsightsAllTimeModel implements Serializable {
```

Comment:
Better to to this in another PR. I need to rename 20+ models and this PR will get dirty to read after that.